### ISSUE TITLE
weavedinstaller Does not prompt for username

### ISSUE THREAD

**Issue by imasonaz** [weaved/installer#3]
When running `weavedinstaller`, I get:
````
Please select from the above options (1-4):
1
weavedinstallerlib Username = REPLACE_USERNAME Password =

Please enter your password:
````
It looks like it meant to prompt for a username, but something went wrong?

I'm running this from: `wget https://github.com/weaved/installer/raw/master/Raspbian%20deb/1.3-06/weavedconnectd-1.3-06i.deb` on a Raspberry Pi A, Wheezy.

**Comment by Digital-Larry** [comment]
Replace "REPLACE_USERNAME" with your username in the script.  Although you are correct, that is in fact a bug.  Thanks for bringing it to my attention.

**Comment by imasonaz** [comment]
Yes. I figured that out. I looked through the code and couldn't figure out why it isn't working, it looks correct here (line 75):
https://github.com/weaved/installer/blob/1c43352579d81424ac1668e0505641828df0c248/Netcomm/weaved-netcomm/weavedsimplelib
and here (line 617 and 636):
https://github.com/weaved/installer/blob/4da03f533e0560ed7f1b0e65943d2891c20bf67d/Ubuntu_deb/weavedconnectd_1.3-06_x86/weavedconnectd_1.3-06_x86/usr/bin/weavedinstallerlib
